Meadowlark-js/services/meadowlark-fastify/src/index.ts

```diff
@@ -5,7 +5,7 @@
 
 import dotenv from 'dotenv';
 import type { FastifyInstance } from 'fastify';
-import { Logger, initializeLogging } from '@edfi/meadowlark-core';
+import { Logger, initializeLogging } from '@edfi/meadowlark-utilities';
 import { buildService } from './Service';
 
 dotenv.config();
```

Comment:
@bradbanister something in this branch broke `dotenv`. When I run `yarn start:local` from the command line, in `\Meadowlark-js\services\meadowlark-fastify`:

* `main` branch: runs fine; I can see that it is reading my .env file because the log output is formatted correctly for `IS_LOCAL=true`
* `RND-336` branch: complains that I don't have a signing key. I manually setup an environment variable again; now it runs, but the log format is as if `IS_LOCAL=false` -- thus it is not reading my `.env` file any more.